docs/src/guides/organizing-stores.mdx

```diff
@@ -260,7 +260,7 @@ Example(
 ...
 ```
 
-A side effect of this is that writing widget tests become easier as well as we'd know what dependencies a widget has. We could then pass, say, mocks (e.g. using the [mockito][2] package] of said dependencies can be passed if required.
+A side effect of this is that writing widget tests become easier as well as we'd know what dependencies a widget has. We could then pass, say, mocks (e.g. using the [mockito][2] package) of said dependencies can be passed if required.
```

Comment:
Can be changed like so for a simpler language:

```
A side effect of this is that writing widget tests become easier, as the dependencies are well known. We could then pass mocks (e.g. using the [mockito][2] package) of said dependencies, if required.
```

Reply:
Thanks. Reworded it a bit now